src/Storages/Kafka/StorageKafka.h

```diff
@@ -112,6 +113,9 @@ class StorageKafka final : public ext::shared_ptr_helper<StorageKafka>, public I
     std::mutex thread_statuses_mutex;
     std::list<std::shared_ptr<ThreadStatus>> thread_statuses;
 
+    /// Handle error mode
+    HandleKafkaErrorMode handle_error_mode;
```

Comment:
I did not find any place to set the mode in StorageKafka.

Reply:
Thanks. I will fix it.